superset/assets/src/components/FilterableTable/FilterableTableStyles.css

```diff
@@ -88,3 +88,7 @@
   white-space: nowrap;
   color: #aaa;
 }
+
+.null-cell {
+  color: #c7c7c7;
```

Comment:
NIT: could use `className='text-muted'` (a bootstrap common class) on the element for a similar effect yet more theme-able approach

Reply:
Thanks for your review @mistercrunch , your recommendations were implemented